components/authz-service/server/v2/project.go

```diff
@@ -568,6 +541,65 @@ func (s *ProjectState) DeleteRule(ctx context.Context, req *api.DeleteRuleReq) (
 	}
 }
 
+const ruleSetEditsPendingStatus = "edits-pending"
```

Comment:
Is there a better place to put this?

Reply:
This might already be an enum somewhere..

Reply:
I think you'll want to use this enum: https://github.com/chef/automate/blob/master/components/authz-service/storage/v2/project_rules_status.go#L11

Reply:
I could not find it in the authz-service only in the automate-gateway.

Reply:
How do I go from the string version to the enum?

Reply:
I think I figured out how to use that.